tests/karate/src/test/java/be/createRollCall/closeRollCall.feature

```diff
@@ -0,0 +1,33 @@
+@env=go,scala
+Feature: Close a Roll Call
+  Background:
```

Comment:
I realize this comment was copy pasted from the preexisting codebase. Nevertheless it seems to me the first line does not make sense and misses the end. Also has the double ## a meaning over the single # one ?

I think it would be good to fix them all.

Reply:
Done